src/main/kotlin/io/github/mojira/arisa/modules/ThumbnailModule.kt

```diff
@@ -20,7 +20,7 @@ class ThumbnailModule(
     private val maxImagesCount: Int
 ) : Module {
 
-    private val log: Logger = LoggerFactory.getLogger(ThumbnailModule::class.java)
+    private val log: Logger = LoggerFactory.getLogger("ThumbnailModule")
```

Comment:
Changed this to be consistent with other classes which also only use the name and not the fully qualified class name.